utils/profile-actions.js

```diff
@@ -28,7 +28,7 @@ export function getRandomEmail() {
 }
 
 export function getSecondPurchaseEmail() {
-  return 'second-purchase-4@mailinator.com'
+  return 'second-purchase-5@mailinator.com'
```

Comment:
Any chance this could be randomized here? Does it have to be a valid email to be accessed at mailinator?

Reply:
This needs to be an email that has already made a purchase so we can test the second purchase flow, and we need access to it in order to authenticate.